modules/042-kube-dns/webhooks/validating/cluster-domain-alias

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+# Copyright 2024 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source /shell_lib.sh
+
+function __config__(){
+  cat <<EOF
+configVersion: v1
+kubernetes:
+  - name: global-module-config
+    apiVersion: deckhouse.io/v1alpha1
+    kind: ModuleConfig
+    executeHookOnEvent: []
+    executeHookOnSynchronization: false
+    keepFullObjectsInMemory: false
+    nameSelector:
+      matchNames:
+      - global
+    jqFilter: |
+      {
+        "publicDomainTemplate": .spec.settings.modules.publicDomainTemplate,
```

Comment:
Please check case when modules is null
It is important because modules and publicDomainTemplate can be ommitted